cypress/e2e/shared/tokens.test.ts

```diff
@@ -198,7 +198,9 @@ describe('tokens', () => {
           .should('exist')
           .click()
       })
-    cy.getByTestID('context-delete-menu--confirm-button').click()
+    cy.getByTestID('context-delete-menu--confirm-button')
+      .should('exist')
```

Comment:
`.should('be.visible')` is better. I've seen flakiness logs where the error was the element was covered, which is a visibility problem.

Reply:
Good idea, I tried using `.should('be.visible')` and it fixed the flakiness. Thanks for the suggestion :)